week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,17 +1,23 @@
 // Add comments to explain what this function does. You're meant to use Google!
+
+//Gets random numberand multiplies it by 10.
 function getNumber() {
     return Math.random() * 10;
 }
 
 // Add comments to explain what this function does. You're meant to use Google!
+
+//cotatinates two variables wi and w2. if a and b replace w1 and w2 we get ab as result.
 function s(w1, w2) {
     return w1.concat(w2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
+    return firstWord.concat(" ", secondWord," ",thirdWord);
     // Write the body of this function to concatenate three words together
     // Look at the test case below to understand what to expect in return 
 }
+console.log (concatenate("code"," your","future"));
```

Comment:
You should remove your `console.log` statements after you're done debugging, otherwise it might confuse other developers for why it's there